docker-compose.yaml

```diff
@@ -1,13 +1,43 @@
 version: "3.4"
 services:
   authz:
-    image: authz
     container_name: authz-service
-    build: /home/authz/git
+    build:
+      context: .
+      dockerfile: Dockerfile.authz
     ports:
-      - 80:5000
+      - "80:5000"
```

Comment:
So we need to place nginx in front of authz service
What we are doing here is mapping 80 port of the host to 5000 port of authz

Reply:
Fixed if I translated the comments correctly))